Processen/CR-PR-API-Beheer.md

```diff
@@ -14,6 +14,32 @@ Het onderstaande diagram beschrijft hoe er wordt om gegaan met issues die in het
+| **2.4** | API Beheer beoordeelt of het issue reeel (de oorzaak ligt bij de API standaard of de Referentie Implementatie) en reproduceerbaar is (ga naar 2.5) of niet (ga naar 2.19). |
+| **2.5** | Zo nodig wordt het issue beter beschreven zodat de kern van het probleem voor iedereen duidelijk is. Tevens wordt aangegeven hoe het probleem opgelost kan worden. Het issue wordt dus ready gemaakt. |
+| **2.6** | Betreft het een kritisch probleem (ga naar 2.7) of niet (ga naar 2.9)? |
+| **2.7** | Levert de oplossing van het probleem een probleem op met de backwardse compatibiliteit (ga naar 2.8) of niet (ga naar 2.9)? |
+| **2.8** | API-Beheer laat het probleem z.s.m. oplossen door een ontwikkelaar. |
+| **2.9** | API-Beheer prioriteert het issue en bepaald of het naar haar idee moet worden opgelost in de volgende release. |
+| **2.10** | Is het belang laag en hoeft het niet mee in de komende sprint (ga naar 2.11), of is het hoog en moet het juist wel mee in de komende sprint (ga naar 2.25) of er is helemaal geen belang en is oplossing niet noodzakelijk (ga naar 2.18)? |
+| **2.11** | Aan de melder van het probleem wordt teruggekoppeld dat het belang van het oplossen van het issue laag is en dat het niet op korte termijn wordt opgelost. |
+| **2.12** | Het issue wordt in de product backlog geplaatst. |
+| **2.13** | API-Beheer analyseert de wens. Zo nodig wordt er meer informatie opgehaald bij de persoon die de wens heeft opgevoerd. Dit kan via diverse media gaan maar altijd wordt hiervan rapportage gedaan in het betreffende GitHub issue. Indien bekend is dat de wens ook bj andere partijen leeft wordt dat ook vermeld. |
+| **2.14** | Zo nodig wordt het issue beter beschreven zodat de kern voor iedereen duidelijk is. |
+| **2.15** | Het issue wordt in de product backlog geplaatst. |
+| **2.16** | De PO prioriteert het issue en bepaald of het naar zijn/haar idee moet worden meegenomen in de volgende release. |
+| **2.17** | Is het belang laag en hoeft het niet mee in de komende sprint (ga naar 2.22), of is het hoog en moet het juist wel mee in de komende sprint (ga naar 2.23) of er is helemaal geen belang en is implementatie niet noodzakelijk (ga naar 2.18)? |
+| **2.18** | Aan de melder van het probleem wordt teruggekoppeld dat er geen belang is bij het oplossen van de bug of implementatie van de wens. | 
+| **2.19** | Is de melder akkoord (ga naar 2.20) of juist niet (ga naar 2.21)? |
+| **2.20** | Het issue wordt afgesloten in GitHub. |
+| **2.21** | Er volgt een escallatie naar .... [**7**] |
+| **2.22** | Aan de indiener van de wens wordt teruggekoppeld dat het belang van de implementatie laag is en dat het niet op korte termijn wordt geimplementeerd. |
```

Comment:
Wordt niet actief teruggekoppeld, staat gewoon in github. Indiener krijgt notificaties.

Reply:
Ik vind dat we het er ons dan wat makkelijk vanaf maken. Als een indiener alle notificaties die binnenkomen moet gaan volgen dan wordt hij/zij knettergek. Notificaties zijn wat mij betreft niet altijd even helder. Ik zou het daarom niet gek vinden om i.i.g. de issues die we afvoeren en de issues die we gaan oplossen waar mogelijk met een mail terug te koppelen. Dat kan in een mail, met daarin al die issues genoemd met de voorgenomen actie (opnemen in komende release of afvoeren), naar alle indieners van de issues in die mail.

Bij een issues dat nog niet wordt opgenomen in de Release backlog vind ik dat je dat expliciet moet vermelden. Daarmee bedoel ik iets anders dan het toevoegen van een label met een lage prio waaruit de indiener maar moet opmaken dat het issue niet de behandeling krijgt die hij graag zou zien. Ook dat beschouw ik als een vorm van actief terugkoppelen.

Reply:
Dit moeten we volgens mij allemaal niet doen. Je mag van een indiener verwachten dat hij zijn issues volgt in Github en ook begrijpt wat het proces waarlangs het wel of niet wordt afgehandeld.

Reply:
Eens met Hugo. Je maakt het jezelf anders te moeilijk.

Notificaties werken zeer goed voor mensen die niet direct betrokken zijn bij het project. Wij werden gek van notificaties omdat ALLES voor ons mogelijk interessant is. Vandaar dat wij het uitzetten en regelmatig de issues/project borden moeten controleren.

Ik krijg bij mij andere open source bijdragen (NLX en Geonovum/KP-API als relevant voorbeeld) juist precies als er iemand reageert op iets wat mijn aandacht heeft.

Reply:
Zoals gezegd in een van de andere conversaties heb ik de terugkoppeling wat neutraler verwoord maar blijven wij nog steeds van mening dat er een actieve terugkoppeling plaats moet vinden.

Reply:
Misschien als toevoeging: ook dit is agile opgezet dus als blijkt dat het teveel werk oplevert of niet goed werkt passen we het aan. Maar ik kan me voorstellen dat het ook voor de contacten met leveranciers en gemeenten in het begin zeker goed is om even terug te koppelen en misschien andere ervaringen of ideeën op te halen etc.